web/css/fonts.css

```diff
@@ -1,21 +1,14 @@
-/* spezia-regular - latin */
+/* open-sans-regular - latin */
 @font-face {
-  font-family: 'Spezia';
+  font-family: 'Open Sans';
   font-style: normal;
   font-weight: 400;
   font-display: swap;
```

Comment:
Preload this font in HTML. using the below line which makes the application faster.

`<link rel="preload" as="font" href="/fonts/custom-font.woff2" type="font/woff2" crossorigin="anonymous">`